src/nimib/renders.nim

```diff
@@ -10,6 +10,23 @@ proc mdOutputToHtml(doc: var NbDoc, blk: var NbBlock) =
 proc highlightCode(doc: var NbDoc, blk: var NbBlock) =
   blk.context["codeHighlighted"] = highlightNim(blk.code)
 
+proc addLineNumbersToHighlightedCode(code: string): string =
+  let nlines = code.splitLines().len
+  # var lineNumbers: seq[string] = @[] # newSeqOfCap(nlines)
+  for i in 1..nlines:
+    ## We have line numbers in the same line than code
+    result.add("""<span class="hljs-comment"> """ & $i & "</span> " & code.splitlines()[i-1] & "\n")
+    ## but we can not copy-paste code without the numbers
+  # The following separates line numbers from code, but display is wrong
+  # result = """ <div>
+  #                 <code>""" & lineNumbers.join("") & """</code>
+  #                 <code>""" & code & """</code>
+  #              </div>
+  #          """
```

Comment:
I think we should wrap the code in a table or a div container so that we can copy-paste code without the numbers.